src/webrtc/call.ts

```diff
@@ -503,6 +509,14 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
         });
     }
 
+    private get hasUserMediaAudioSender(): boolean {
+        return this.transceivers.has(getTransceiverKey(SDPStreamMetadataPurpose.Usermedia, "audio"));
+    }
+
+    private get hasUserMediaVideoSender(): boolean {
+        return this.transceivers.has(getTransceiverKey(SDPStreamMetadataPurpose.Usermedia, "video"));
+    }
```

Comment:
This seems to only be checking for `transceiver`s not `sender`s - that doesn't feel right

Reply:
True, it should probably be checking the sender